tests/e2e/support/fixtures.ts

```diff
@@ -0,0 +1,34 @@
+import { test as base } from '@playwright/test'
+
+import { type Fixture, deployFixture } from './deploy-to-netlify'
+
+/* eslint-disable-next-line @typescript-eslint/no-empty-interface */
+interface TestFixtures {}
+interface WorkerFixtures {
+  serverlessSite: Fixture
+  edgeSite: Fixture
+}
+
+export const test = base.extend<TestFixtures, WorkerFixtures>({
+  serverlessSite: [
+    async ({}, use) => {
+      // Set up the fixture
+      const fixture = await deployFixture('serverless-site')
+      // Use the fixture value in the test
+      await use(fixture)
+      // Cleanup goes here
+    },
+    { scope: 'worker' },
+  ],
+  edgeSite: [
+    async ({}, use) => {
+      // Set up the fixture
+      const fixture = await deployFixture('edge-site')
+      // Use the fixture value in the test
+      await use(fixture)
+      // Cleanup goes here
```

Comment:
Missing cleanup? Or are you just indicating this for future authors?

Reply:
It was a hint for devs, yes. It's kind of a weird API, so I thought I'd make it clear. I wouldn't mind removing it.

Reply:
Maybe just make the comment more explicit